src/cpp/src/utils.cpp

```diff
@@ -260,6 +260,23 @@ void slice_matmul_statefull_model(std::shared_ptr<ov::Model> model) {
     }
 }
 
+template <typename T>
```

Comment:
template function cannot be defined in .cpp file, how does it work?

Reply:
I am not quite sure how it works. At first, it didn't work (error at linking), then i added this instantiations and it worked
```
template void read_rt_info<int64_t>(std::shared_ptr<ov::Model>&,  const char*, int64_t&);
template void read_rt_info<std::string>(std::shared_ptr<ov::Model>&,  const char*, std::string&);
```

But since templates are compile time feature. I agree, probably it's better to move them into headers.

Reply:
> added this instantiations and it worked

it's also a working approach